File: ConexaoVagas API/Domains/Candidatura.cs
using System;
using System.Collections.Generic;

namespace ConexaoVagasAPI.Domains
{
    public partial class Candidatura
    {
        public int IdCandidatura { get; set; }
        public int IdCandidato { get; set; }
        public int IdVaga { get; set; }
        public string Curriculo { get; set; }
        public bool Visualizado { get; set; }
        public DateTime DataCriado { get; set; }

        public Candidato IdCandidatoNavigation { get; set; }
        public Vaga IdVagaNavigation { get; set; }
    }
}

File: ConexaoVagas Front/src/models/candidatura.ts
import { Candidato } from "./candidato";
import { Vaga } from "./vaga";

export class Candidatura {
    idCandidatura?: number;
    idCandidato?: number;
    idVaga?: number;
    curriculo?: string;
    visualizado?: boolean;
    dataCriado?: string;
    idCandidatoNavigation?: Candidato = undefined;
    idVagaNavigation?: Vaga = undefined;
}